components/Article.js

```diff
@@ -0,0 +1,17 @@
+import styled from "styled-components";
+
+export default function StyledArticle(props) {
```

Comment:
Hi, maybe you can destructure die children 
```suggestion
export default function StyledArticle({children}) {
return <Article>{children}</Article>
```